framework/src/modules/network/defaults/config.js

```diff
@@ -141,6 +131,7 @@ const defaultConfig = {
 		},
 		wsMaxPayload: {
 			type: 'integer',
+			maximum: 3048576,
```

Comment:
why do we need to enforce maximum here?

Reply:
Exceed this limit will cause that you will be banned from the network.